nova/virt/vmwareapi/vmops.py

```diff
@@ -819,10 +826,6 @@ def spawn(self, context, instance, image_meta, injected_files,
                                             extra_specs,
                                             metadata)
 
-        # Cache the vm_ref. This saves a remote call to the VC. This uses the
```

Comment:
I think, it would be good to keep the old cache updates still in place.
It should do no harm to remove / add the values twice.